searx/engines/community_scripts_proxmoxve.py

```
@@ -155,13 +231,67 @@ def search(query: str, params: "RequestParams") -> EngineResults:  # pylint: dis
+    slugs_list = CACHE.get("script_slugs_list")
+    
+    if isinstance(slugs_list, list) and slugs_list:
+        _logger.debug("Attempting to retrieve %d scripts from individual cache entries.", len(slugs_list))
+        temp_scripts = []
+        retrieval_successful = True
+        for slug in slugs_list:
+            cached_script = CACHE.get(f"script_{slug}")
+            if cached_script:
+                try:
+                    decompressed_script = zlib.decompress(cached_script)
+                    script = pickle.loads(decompressed_script)
```

Ruff (0.15.1)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)